cmd/helm/install.go

```diff
@@ -231,6 +231,14 @@ func (o *installOptions) run(out io.Writer) error {
 		return err
 	}
 
+	chartType := chartRequested.Metadata.Type
```

Comment:
I see these same tests multiple times in this PR. Can we just make a function?

Reply:
Good idea. Function added.